### ISSUE TITLE
Giscus comments

### ISSUE THREAD

**Issue by bravo-kernel** [pester/docs#251]
### Checklist

- [X] Issue has a meaningful title
- [X] I have searched the existing issues. See [issues](https://github.com/pester/docs/issues)

### Summary of the request

We could consider adding Giscus comments (built on Github discussions) for pages:

- Matches our audience
- Upvotes, reactions, etc.
- Easily moderated via github interface
- Could lead to more interaction

Additional info:

- https://github.com/giscus/giscus (make sure to open the https://gisqus.app)
- https://m19v.github.io/blog/how-to-add-giscus-to-docusaurus

### Suggested section

None

**Comment by fflaten** [comment]
Looks like a cool solution. Comments is a bit uncommon for docs-sites though, isn't it?

_Off-topic: In your example, why does it look like you're talking to yourself?_ :)

**Comment by bravo-kernel** [comment]
I agree that it is a bit uncommon but I can imagine it making the hurdle to comment, ask, etc. much lower.

The reason you are seeing me talking to myself (online and in real-life now) is... that those comments were imported from Disqus (hover on a username).

**Comment by nohwnd** [comment]
Upvote on this, azdo has comments for wiki and it is very useful when the page becomes outdated and people post tips on how to actually do the thing the page describes.

**Comment by fflaten** [comment]
Sounds great 👍 
@bravo-kernel still working good  on your blog. No issues after the theme-fix? Removing spam etc also works fine?

Usually I'd say GitHub-only would limit a comment-solution, but as it's already required to make a docs-issue it's fine.

**Comment by bravo-kernel** [comment]
No issues at all and even though spam is non-existent, annoying responses can always be deleted in Github Discussions.
